src/pages/en/core-concepts/astro-components.md

```diff
@@ -125,22 +125,45 @@ They can be used to style your components, and all style rules are automatically
 
 ### Client-Side Scripts
 
-To send JavaScript to the browser without [using a framework component](/en/core-concepts/component-hydration) (React, Svelte, Vue, Preact, Solid...) you can use a `<script>` tag in your Astro component body.
+To send JavaScript to the browser without [using a framework component](/en/core-concepts/framework-components) (React, Svelte, Vue, Preact, SolidJS, AlpineJS, Lit...) you can use a `<script>` tag in your Astro component template and send JavaScript to the browser that executes in the global scope.
+
 
 ```astro
----
-// Your component script here!
----
 <script>
-  document.querySelector('h1').style.color = 'red';
+  // Will be rendered into the HTML exactly as written!
+  // ESM imports will not be resolved relative to the file.
 </script>
 
-<h1>Hello, world!</h1>
+<script hoist type="module">
+  // Processed! Bundled! ESM imports work, even to npm packages.
+</script>
+```
+#### Loading External Scripts
+
+**When to use this:** If your JavaScript file lives inside of `public/`.
+
+Note that this approach skips the JavaScript processing, bundling and optimizations that are provided by Astro when you use the `import` method described below. 
+
+```astro
+// absolute URL path
+<script src="/some-external-script.js"></script>
+```
+#### Using Hoisted Scripts
```

Comment:
I don't think this section is needed, it's basically repeating the "Client-Side Scripts" section above.

Reply:
My concern if we take this out is that then there is no explicit example showing the import method. (The pseudo code examples simply show the difference in principle.)

So, I'm going to make a judgment call and leave it in. Happy to revisit if you have a better example to use here, but I do think each should explicitly have an example to show the differences in syntax.